index.html

```diff
@@ -3879,8 +3940,7 @@ <h2 id="GainNode">
           </dt>
           <dd>
             <p>
-              Represents the amount of gain to apply. Its default
-              <code>value</code> is 1 (no gain change). This parameter is
```

Comment:
I'm ok with adding defaults to the dictionary members, but I'm not sure I want it removed from the attribute definition too.  Now the reader has to magically know to go look at the dictionary definitions to get the default value.